src/components/util/hexToRgb.ts

```diff
@@ -0,0 +1,12 @@
+
+
+export default function hexToRgb(hex: string, opacity: number): string {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+
+  return result
+    ? `rgba(${parseInt(result[1], 16)}, ${parseInt(result[2], 16)}, ${parseInt(
```

Comment:
improvement, lets create a const and reuse it

`const BASE_16 = 16`
`rgba(${parseInt(result[1], BASE_16)}`

Reply:
@mad-raccoon done

Reply:
did you push ? i dont see the changes